stdlib/public/Concurrency/ExecutorAssertions.swift

```diff
@@ -106,11 +106,24 @@ extension Actor {
       return
     }
 
+    // _taskIsCurrentExecutor was designed for Builtin.Executor
+    // OLD: let expectationCheck = _taskIsCurrentExecutor(self.unownedExecutor.executor)
+
+    // the swift_task_isOnExecutor that we're not using was designed for <SE: SerialExecutor>...
+    // We could abuse this not-used entrypoint from always-emit-into-client `preconditionIsolated` funcs
+    // but... how do we call it...?
+    // _taskIsOnExecutor(???)
+    // FIXME: how form such call from Swift:
+    //    HeapObject *executor,
+    //    const Metadata *selfType,
+    //    const SerialExecutorWitnessTable *wtable
+
+    // NOTE: This method will CRASH in new runtime versions,
+    // if it would have previously returned `false`.
+    // It will call through to SerialExecutor.checkIsolated` as a last resort.
     let expectationCheck = _taskIsCurrentExecutor(self.unownedExecutor.executor)
```

Comment:
This is the PoC, so we're crashing here in this PoC PR. But we cannot keep it this way, we must start calling the other entry point to the runtime.